src/libpgmoneta/wal.c

```diff
@@ -285,6 +286,11 @@ pgmoneta_wal(int srv, char** argv)
                            pgmoneta_log_error("Could not create or open WAL segment file at %s", d);
                            goto error;
                         }
+
+                        current_wal_filename = pgmoneta_append(current_wal_filename, filename);
+                        current_wal_filename = pgmoneta_append(current_wal_filename, ".partial");
+                        strcpy(config->servers[srv].current_wal_filename, current_wal_filename);
```

Comment:
I feel you need to initialize it to an empty string, in case we are not streaming WAL due to network error or server down. @jesperpedersen What do you think?

Reply:
The server struct gets initalized.

But, `current_wal_filename` needs to get `free()`d, and `configuration.c` need updates for reload

Reply:
> The server struct gets initalized.

But srv.current_wal_filename is not, is it? We need to add it to where the struct gets initialized.

Reply:
Oh yeah, that'll do.